updates/testing.json

```diff
@@ -1,7 +1,7 @@
 {
   "stream": "testing",
   "metadata": {
-    "last-modified": "2019-09-09T10:01:35Z"
+    "last-modified": "2019-09-10T13:49:17+00:00"
```

Comment:
the instructions say `date -u -Is` which gives me this format, which is not consistent with what we had in the past

Reply:
In cosa, we use `date -u +%Y-%m-%dT%H:%M:%SZ`.